src/document-submit.js

```diff
@@ -28,11 +27,7 @@ import {
  * @param {!./service/ampdoc-impl.AmpDoc} ampdoc
  */
 export function installGlobalSubmitListenerForDoc(ampdoc) {
-  return getServiceForDoc(ampdoc, 'submit', ampdoc => {
-    ampdoc.getRootNode().addEventListener(
-        'submit', onDocumentFormSubmit_, true);
-    return {};
-  });
+  ampdoc.getRootNode().addEventListener('submit', onDocumentFormSubmit_, true);
```

Comment:
So it looks like this was done to only register the event listener if a form requests the submit service.